avocado_vt/vt_agent/src/avocado_vt/agent/core/__init__.py

```
@@ -0,0 +1,9 @@
+"""
+The `core` package for the Avocado VT Agent.
+
+This package provides the central functionalities of the agent, including
+the RPC server, service management, data directory handling, and logging setup.
+"""
+
+# pylint: disable=E0611
+from avocado_vt.agent.core import data_dir, logger, rpc
```

Ruff (0.12.2)

`avocado_vt.agent.core.data_dir` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)